packages/svelte-ux/src/lib/components/SelectField.svelte

```diff
@@ -49,13 +60,17 @@
   } = {};
   const theme = getComponentTheme('SelectField');
 
+  let fieldClasses: ComponentProps<TextField>['classes'];
+  $: fieldClasses = typeof(classes.field) === "string" ? { root: classes.field } : classes.field;
+
   // Menu props
   export let placement: Placement = 'bottom-start';
   export let autoPlacement = true;
   export let matchWidth = true;
   export let resize = true;
   export let disableTransition = false;
   export let menuProps: ComponentProps<Menu> | undefined = undefined;
+  export let optionsMode: "list" | "menu" = "menu";
```

Comment:
Maybe call it `inlineOptions` since we can’t really support more than 2 modes (is always either `Menu` and passed props, or `undefined` which ignores the props)

Reply:
Made this change.